test/board_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule BoardTest do
+  use ExUnit.Case
+
+  test "populates initial board" do
+    assert Board.setup_initial_board() ===
+      %{1 => "1", 2 => "2", 3 => "3", 4 => "4", 5 => "5", 6 => "6", 7 => "7", 8 => "8", 9 => "9"}
+  end
+
+  test "replaces the 2 with an X" do
+    board = %{1 => "1", 2 => "2", 3 => "3", 4 => "4", 5 => "5", 6 => "6", 7 => "7", 8 => "8", 9 => "9"}
+
+    board = Board.place_a_symbol(board, "2", "X")
+
+    expected_board = %{1 => "1", 2 => "X", 3 => "3", 4 => "4", 5 => "5", 6 => "6", 7 => "7", 8 => "8", 9 => "9"}
```

Comment:
I wonder if Credo would complain about this:
```elixir
expected_board = %{
  1 => "1", 2 => "X", 3 => "3",
  4 => "4", 5 => "5", 6 => "6", 
  7 => "7", 8 => "8", 9 => "9"
}
```

Reply:
I can check! It does look nicer, would this un-approve the changes?

Reply:
Credo was all good with it so I changed it, I like it better! Thank you 😄

Reply:
Yea sorry for the confusion. I wasn't sure if this is a valid format for Credo so this won't affect the approval! Thanks for checking and making the change Jon!